build/iptablesManager/iptablesManager-ds.yaml

```diff
@@ -0,0 +1,69 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: cloud-iptables-manager
+  namespace: kubeedge
+  labels:
+    app: iptables-manager
+spec:
+  selector:
+    matchLabels:
+      app: iptables-manager
+  template:
+    metadata:
+      labels:
+        app: iptables-manager
+    spec:
+      serviceAccount: iptables-manager-sa
+      hostNetwork: true
+      containers:
+        - name: iptables-manager
+          command: ['iptables-manager']
+          image: kubeedge/iptables-manager:latest
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: FORWARD_PORT
+              value: "10003"
+          securityContext:
+            capabilities:
+              add:
+                - NET_ADMIN
+                - NET_RAW
+---
```

Comment:
Do we need to add the node affinity to make the pods always running on master node?

Reply:
I think it needs to keep on the k8s nodes. Cannot deploy it on the edge side.  For the case with the worker node on the k8s platform, it also can be forwarded to the <cloudcore VIP>: <internal cloudcore streamport> if we expose the service type as nodeport.

Reply:
I will add node affinity to keep it not scheduling to the edge side.